core/templates/dev/head/pages/admin/activities_tab/AdminDevModeActivitiesTabDirective.js

```diff
@@ -104,6 +104,24 @@ oppia.directive('adminDevModeActivitiesTab', [
           }
         };
 
+        $scope.generateDummyExplorations = function() {
+          // Generate dummy explorations with random title.
+          AdminTaskManagerService.startTask();
+          $scope.setStatusMessage('Processing...');
+          $http.post(ADMIN_HANDLER_URL, {
+            action: 'generate_dummy_exploration',
+            dummy_exp_count: $scope.DummyExplorationCount,
```

Comment:
Should be $scope.dummyExplorationCount, $scope.dummyExplorationPublishCount -- and these vars should be initialized in the main controller.